online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/OnlineBankingCZView.kt

```diff
@@ -109,6 +114,29 @@ class OnlineBankingCZView @JvmOverloads constructor(
         binding.textInputLayoutOnlineBanking.isEnabled = enabled
     }
 
+    private fun styleTermsAndConditions(string: String) {
```

Comment:
Can we extract this method to a a common module in case we need it later? Also can you check what happens if the merchant overrides this string with something that doesn't have `%#` `#%`? Maybe add some tests around it to be safe?

Reply:
about extracting the method I agree with you on this, but if they override this strings I guess they will reimplement their own `initLocalizedStrings` right?